keyboards/fa65/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "fa65",
+    "manufacturer": "hu",
+    "url": "",
```

Comment:
Prretty sure this should link to a store where to buy/guide on how to build/whatever